File: letter/ru/Е.py
from common import *

expr = Eq(
    div(
        (Abs(y) - 1)
        * y
        * (x + S(1) / 2)
        * sqrt((x + S(1) / 2 + eps) * (1 + eps - Abs(y))),
        sqrt(S(1) / 2 - x)
    )
    ,
    0
)
size = 1

File: letter/ru/Е.js
import { Letter } from '../../Letter.js';

const expr = '(abs(y) - 1) * y * (x + 1/2) * sqrt((x + 1/2 + 1/10) * (1 + 1/10 - abs(y))) / sqrt(1/2 - x)';

export const letter = new Letter(expr, 1);
